zenodo/config.py

```diff
@@ -935,7 +935,7 @@ def _(x):
             'application/x-bibtex': (
```

Comment:
Search also for `ZenodoDataCite31Serializer`, `DataCite31Serializer`, and `DataCiteSchemaV1` which should all be removed. This will probably break tests also (so you might have to remove some of the tests as well.